### ISSUE TITLE
If inside an 'and' expression, a property contains the string 'and' (like 'brand'), it crashes. 

### ISSUE THREAD

**Issue by rodri042** [TossShinHwa/node-odata#49]
In this case, the parser believes that I am concatenating another **and** expression. My model is:

```
Product
    name: String
    code: String
    category: String
    brand: String
```

The query:
`?$filter=name eq '600ml Bottle' and brand eq 'Pepsi'`
...results in:
`500 - Syntax error at 'br'.`

If I rename the property to **bran** or something, it works.

**Comment by TossShinHwa** [comment]
Wow, it's seems some thing wrong with regex for parse querystring, I will check it later.

thx.

**Comment by TossShinHwa** [comment]
Fixed in v0.7.9
